### ISSUE TITLE
Can't import SoF 2 models

### ISSUE THREAD

**Issue by Nitestokeh** [mrwonko/Blender-2.6-Ghoul-2-addon#9]
When I try to import .glm, it says "File not found! (no .gla?)" Was this script intended only for JA or am I doing something wrong?

**Comment by spoilt-exile** [comment]
Unfortunately this plugin usage is a bit complicated. You must use blender 2.6.3 but not above 2.6.x; You must extract JA game asset pk3 entirely and define it location as parametr during import.

**Comment by Nitestokeh** [comment]
Thanks for such quick reply. I have everything extracted and location defined, yet something's still wrong.

**Comment by spoilt-exile** [comment]
AFAIK there is some limitation in extracted folder name (!). It should be just 'gamedata' or something else. I'll check this at evening on my home pc. Plugin works fine there.

**Comment by mrwonko** [comment]
Yeah, it's built with Jedi Academy in mind. If your paths are not relative to a subfolder of GameData that also contains the file you're importing (e.g. you're importing ```C:/JKA/GameData/base/models/mymodel.glm``` which references ```models/myskeleton.gla```, which will be taken to be relative to ```C:/JKA/GameData/base``` based on the ```/GameData/``` in the path) you'll have to set the *basepath* property on import.

So if you're importing ```C:/SOF2/models/mymodel.glm``` you'll want to set the *basepath* to ```C:/SOF2/```.

This plugin is part of a [Plugin Suite](http://jkhub.org/files/file/1413-blender-264-jedi-academy-plugin-suite/) which also contains some documentation that you may want to read.
